src/components/display/ComponentDelegate.tsx

```diff
@@ -19,6 +24,16 @@ export const ComponentDelegate = ({ component }: { component: ComponentObject })
 			return <InputComponent {...component.args} />
 		case 'button':
 			return <ButtonComponent {...component.args} />
+		case 'link':
+			return <LinkComponent {...component.args} />
+		case 'checkbox':
+			return <CheckboxComponent {...component.args} />
+		case 'badge':
+			return <BadgeComponent {...component.args} />
+		case 'progress':
+			return <ProgressBarComponent {...component.args} />
+		case 'dropdown':
+			return <DropdownComponent {...component.args} />
```

Comment:
Needs action popover added

Reply:
I skipped it cause we don't have it added in the ComponentObject. Can you please check?

Reply:
Fixed in display-api@0.2.8